src/app/storage.js

```diff
@@ -24,3 +24,31 @@ export function storageAvailable(type) {
     );
   }
 }
+
+export function getLocalSetting(key) {
+  if (storageAvailable("localStorage")) {
+    return window.localStorage.getItem(key);
+  }
+}
+
+export function setLocalSetting(key, value) {
+  if (storageAvailable("localStorage")) {
+    window.localStorage.setItem(key, value);
+  }
+}
+
+export function rehydrateLocalSettings() {
+  let checkAutoplay = getLocalSetting("autoplay");
+
+  if (typeof checkAutoplay === "undefined") {
+    checkAutoplay = true; // default to true
+  } else {
+    checkAutoplay = checkAutoplay === "true";
```

Comment:
I might be tracing it wrong, but I think it's being stored as a bool... ?

Reply:
So localStorage stores everything as a string. We need to convert all non-string values.